manifests/global_defs.pp

```diff
@@ -66,6 +72,8 @@
   $enable_traps                                   = undef,
   Optional[Boolean] $vrrp_higher_prio_send_advert = undef,
   Optional[Integer] $vrrp_garp_lower_prio_repeat  = undef,
+  Optional[Integer] $vrrp_garp_master_refresh     = undef,
```

Comment:
do you know which integer values are the minimum and maximum? If so, we can enforce them as well.

Reply:
It's not mentioned in the documentation, only that the default is 0 (no refreshing). 
Looking at https://github.com/acassen/keepalived/blob/245e2ac65dade95f5ccbecb4b1d9d478966418af/keepalived/core/global_parser.c#L680 it is set to UINT_MAX, but I am no C expert.